Makefile

```diff
@@ -25,3 +25,7 @@ tests:
 .PHONY: benchmarks
 benchmarks:
 	make -C tests benchmarks
+
+.PHONY: system-tests
```

Comment:
Done.  No doubt we'll need to move files and directories around later when we add more system tests.